src/Control/Monad/Codensity.hs

```diff
@@ -156,8 +141,13 @@ instance (v ~~ v', Alternative v')
 -- e.g. @'Codensity' ((->) s)) a ~ forall r. (a -> s -> r) -> s -> r@
 -- could support a full complement of @'MonadState' s@ actions, while @(->) s@
 -- is limited to @'MonadReader' s@ actions.
+#if __GLASGOW_HASKELL__ >= 710
```

Comment:
What is this CPP doing here? `kan-extensions` now [requires GHC 8.4 or later](https://github.com/ekmett/kan-extensions/pull/77), so I can't fathom why this is needed. (Did you make sure to base your changes off of the most recent commit in `master`?)